cmd/validate/image.go

```diff
@@ -170,6 +180,58 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 				SubjectRegExp: data.certificateIdentityRegExp,
 			}
 
+			// Check if policyConfiguration is a git url, if so then we need to download the config
+			if strings.HasPrefix(data.policyConfiguration, "git::") || strings.HasPrefix(data.policyConfiguration, "github.com/") || strings.HasPrefix(data.policyConfiguration, "https://github.com/") {
```

Comment:
Since we're adding more logic here this and the branch below should be responsibility of `github.com/enterprise-contract/ec-cli/internal/policy/source`

Reply:
Done. (In incoming revision).